pkg/server/status.go

```diff
@@ -1164,6 +1165,7 @@ func (s *statusServer) handleVars(w http.ResponseWriter, r *http.Request) {
 		log.Error(r.Context(), err)
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 	}
+	telemetry.Count("web-ui.prometheus.vars")
```

Comment:
similar comment here. Many monitoring tools besides crdb's own web UI use this endpoint.

My general recommendation would be to create feature names that describe exactly what's being measured, now how you think this control path is being taken from.